test/browser/browser_asrouter_targeting.js

```diff
@@ -285,6 +285,18 @@ add_task(async function checkFrecentSites() {
   TopFrecentSitesCache.expire();
 });
 
+add_task(async function check_firefox_version() {
+  const message = {id: "foo", targeting: "firefoxVersion > 0"};
```

Comment:
What I considered is `undefined > 0 // false` if there is an issue and the getter does not return a value. Not sure how to make the test more strict while still working across firefox versions.

Reply:
I think this is fine, what you really want to test here is probably that we're getting an integer anyway (AppConstants.MOZ_APP_VERSION is presumably pretty safe)